<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtvplugin_grid</class>
 <widget class="QWidget" name="qtvplugin_grid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grid</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/Brush_Ruler_Alt.png</normaloff>:/icons/Brush_Ruler_Alt.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_QTV_measure">
       <property name="text">
        <string>Ruler Active</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/Brush_Ruler_Alt.png</normaloff>:/icons/Brush_Ruler_Alt.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_qtv">
     <property name="title">
      <string>Mark</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_QTV_markcmd">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton_QTV_latfirst">
          <property name="text">
           <string>LatLon</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_QTV_lonfirst">
          <property name="text">
           <string>LonLat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_QTV_2">
          <property name="text">
           <string>,West spliter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_QTV_west_spliter"/>
        </item>
        <item>
         <widget class="QLabel" name="label_qtv">
          <property name="text">
           <string>South spliter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_QTV_south_spliter"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>37</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="combox_type"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_QTV_add_mark">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../qtvplugin_geomarker/resources.qrc">
            <normaloff>:/icons/Add to favourites.png</normaloff>:/icons/Add to favourites.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_QTV_clear">
          <property name="text">
           <string>Clear</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/Delete-1.png</normaloff>:/icons/Delete-1.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_QTV_clear_all">
          <property name="text">
           <string>Clear All</string>
          </property>
          <property name="icon">
           <iconset resource="../qtvplugin_geomarker/resources.qrc">
            <normaloff>:/icons/delete-21.png</normaloff>:/icons/delete-21.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_QTV_5">
     <property name="text">
      <string>Cursor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit_QTV_cursor">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_QTV_4">
     <property name="text">
      <string>Results</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit_QTV_RES">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qtvplugin_geomarker/resources.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
